internal/api/cluster_installation.go

```diff
@@ -359,3 +364,240 @@ func handleRunClusterInstallationMattermostCLI(c *Context, w http.ResponseWriter
 	w.WriteHeader(http.StatusOK)
 	w.Write(output)
 }
+
+// handleMigrateClusterInstallations responds to Post /api/cluster_installation/migrate.
+func handleMigrateClusterInstallations(c *Context, w http.ResponseWriter, r *http.Request) {
+	mcir, err := model.NewMigrateClusterInstallationRequestFromReader(r.Body)
+	if err != nil {
+		c.Logger.WithError(err).Error("failed to decode cluster migration request", mcir)
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	c.Logger = c.Logger.WithField("migration request for cluster installation", mcir)
```

Comment:
Not sure if we want to include whole request as a logger field as it is a little clunky:
```
... migration request for cluster installation="{ 12345 67899 false false}"
```
Maybe adding two fields with `source-cluster-id` and `target-cluster-id` would be slightly cleaner.

Reply:
Done. Thanks